sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -828,9 +855,41 @@ public void deleteDataset(String projectId, String datasetId)
         List<ValueInSingleWindow<T>> failedInserts,
         ErrorContainer<T> errorContainer,
         boolean skipInvalidRows,
-        boolean ignoreUnkownValues,
+        boolean ignoreUnknownValues,
         boolean ignoreInsertIds)
         throws IOException, InterruptedException {
+      return insertAll(
```

Comment:
Looks like now we have four variants of `insertAll`:
```
  a) public insertAll (with successfulRows) 
  b) public insertAll (without successfulRows)
  c) protected insertAll (with successfulRows)
  d) protected insertAll (without successfulRows)
// b calls a, d calls c, a calls c
```
If these methods are mostly for internal use, I would prefer to have just:
```
  a) public insertAll (with successfulRows)   
  b) protected insertAll (with successfulRows)
// a calls b
```
and explicitly put `null` for all existing call-sites of `a` and `b`.

Reply:
Done